python/cuml/test/test_metrics.py

```diff
@@ -152,3 +152,35 @@ def test_rand_index_score(name, nrows):
     cu_score_using_sk = sk_ars(y, cu_y_pred)
 
     assert array_equal(cu_score, cu_score_using_sk)
+
+
+@pytest.mark.parametrize('datatype', [np.float32, np.float64, np.int])
+@pytest.mark.parametrize('use_handle', [True, False])
+def test_homogeneity_score(datatype, use_handle):
+    def score_labeling(ground_truth, predictions):
+        a = np.array(ground_truth, dtype=datatype)
+        b = np.array(predictions, dtype=datatype)
+
+        a_dev = cuda.to_device(a)
+        b_dev = cuda.to_device(b)
+
+        handle, stream = get_handle(use_handle)
+
+        return cuml.metrics.homogeneity_score(a_dev, b_dev, handle=handle)
```

Comment:
probably we want to compare against sklearn's metrics as a nice reference implementation?

Reply:
Agreed and done.